2017-10/agenda.md

```diff
@@ -0,0 +1,46 @@
+# Node.js Collaborator Summit, 2017-10-06 - 2017-10-07, Vancouver, BC, Canada
+
+**This is a DRAFT proposed agenda**
+
+## 2017-10-06 Day One
+
+|    Time Slot    | Room |      Session     |        Description       |
+| --------------- | ---- | ---------------- | ------------------------ |
+|  9:00a - 11:00a | TBD  | Core Plenary     | All-Collaborator Plenary |
+| 11:00a - 12:00p | TBD  | TBD              | TBD                      |
+| 12:00p - 1:00p  | TBD  | Lunch            | Lunch                    |
+|  1:00p - 4:00p  | TBD  | WG Sessions      | Working Group Sessions   |
+|  1:00p - 3:00p  | TBD  | CommComm Plenary | CommComm Plenary Session |
+|  4:00p - 5:00p  | TBD  | TBD              | TBD                      |
+
+## 2017-10-07 Day Two
```

Comment:
@jasnell Is there a confirmed venue for Day Two?

Reply:
As far as I know, @hackygolucky would need to confirm